auth_proxy/proxy/flask.py

```diff
@@ -36,30 +36,32 @@ class FlaskClient(Client):
         'Procedure',
     ]
 
+    SECURITY_ARG_NAME = '_security'
```

Comment:
Rather than pulling out these one-line functions and overriding them in the `UnsecureFlaskClient`, you might consider abstracting all of the security-related logic into a single `enforce_security` function that requests can call.

Reply:
I think it looks cleaner and less tedious like you suggested.